contributing.md

```diff
@@ -0,0 +1,167 @@
+# Contributing
+
+> You want to contribute to the website? Welcome!
+> Our website runs off of VuePress. Here you will find contribution resources and advice.
+
+If you have any questions, you can ask them in #website-dev channel in our Discord.
+
+ * [Preparing the development environment](#preparing-the-development-environment)
+ * [Guidelines](#guidelines-1)
+ * [Localization](#localization)
+
+## Preparing the development environment
+### Fork the project
+* Create your own fork of the repository by choosing the **Fork** option at the top right of the website github repository.
+* Create a new branch before working on your new fork. The method of branching depends on the Git CLI/GUI that you use, reference your software manual, or contact us if you need help with this step.
+* Once on your own branch, you're ready to start making improvements to the website
+
+> You can use a git client like [GitHub Desktop](https://desktop.github.com/) to manage your local repository.
+
+When your done submit a pull request. This can be done by navigating to the main page of the [website git repository](https://github.com/Paperback-iOS/website) and selecting 'pull request' at the top.  You may have to click the button titled **compare across forks**. Ensure that the base repository belongs to Paperback-iOS, whereas the head is your fork, and your branch.
+
+### Running the website
+> When you make modifications to the site, you should validate your changes and formatting yourself first.
+
+See [Building local enviroment](https://github.com/Paperback-iOS/website#building-local-enviroment)
```

Comment:
You're right, this would be better.
To be honest I'm a little lost with your suggestion... is this better than running `npm run serve`?

An other question, could `Node.js` be the right spelling?
```suggestion
**First time setup**
1. Open up a terminal window and navigate to your project root
1. Run the following command: `npm install`
  > You must have Node.js installed on your development environment, otherwise npm will not be a valid command
**Running the website**
1. Navigate to the project root in a terminal
1. Run the following command: `npx vuepress dev src`
  > npx is your node package executor. This allows you to run VuePress without installing it globally with npm or yarn.
```

Reply:
Yeah, in hindsight your spelling is correct.
npm serve as a command may work just fine! If you’d rather swap it to that, that’s probably reasonable. I recall I had issue with that for some reason, so I listed out the full command.

My suggestion is simply what npm serve actually is doing.